pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -1117,18 +1185,20 @@ func (s *server) Upsert(ctx context.Context, req *payload.Upsert_Request) (loc *
 		loc, err = s.Update(ctx, &payload.Update_Request{
 			Vector: req.GetVector(),
 			Config: &payload.Update_Config{
-				Timestamp: req.GetConfig().GetTimestamp(),
+				Timestamp:            req.GetConfig().GetTimestamp(),
```

Comment:
is there any chance that the req or req.GetConfig() is nil?

Reply:
It's protocol buffer auto generated code, req.GetConfig() never panics if req is nil so req.GetConfig().GetTimestamp() never panics if req is nil or req.GetConfig()  is nil, please read code.